apps/chat/src/backend/serverless/template.yaml

```diff
@@ -1599,12 +1599,16 @@ Resources:
         PriceClass: PriceClass_100
         Restrictions:
           GeoRestriction:
-            RestrictionType: whitelist
-            Locations:
-              - US
-              - JP
-              - AU
-              - FR
+            RestrictionType: none
+        CustomErrorResponses:
+          - ErrorCachingMinTTL: 300
+            ErrorCode: 404
+            ResponseCode: 200
+            ResponsePagePath: /chat.html

```

Comment:
So here for 404, 403s we show the chat.html page. What does it look like and is that a static show page? Having a screenshot in description would help :)

Also, description mentions a S3 bucket fix and I also see the commit but no file change for it in commit?